lib/buyers/models.py

```diff
@@ -27,6 +27,10 @@ class Buyer(Model):
     new_pin = HashField(blank=True, null=True)
     needs_pin_reset = models.BooleanField(default=False)
     email = AESField(blank=True, null=True, aes_key='buyeremail:key')
+    # Because the email field is encrypted we can't do lookups in mysql on it.
+    # This allows us to use a field for lookups, without exposing anything in
+    # the database.
+    email_hash = ConsistentHashField(blank=True, null=True)
```

Comment:
It may be pedantic so feel free to ignore this but I think this would be better named `email_sig` using `ConsistentSigField` because technically it is not a hash and if you were to actually hash the emails [Ben Adida would be sad](http://benlog.com/2008/06/19/dont-hash-secrets/). I am also suggesting this because I generally dislike naming things according to their implementation which is why I am not suggesting to name it `email_hmac` :)